packages/build/src/log/messages/core.js

```diff
@@ -46,13 +46,15 @@ const logLingeringProcesses = function (logs, processList) {
     logs,
     THEME.errorLine(`
 ** WARNING **
-There are some lingering processes even after the build process finished:
+The build completed successfully but the following processes were accidentally
+left running.
```

Comment:
Some suggested edits below. I'm not sure whether the accidental part is important, but this is a bit more active.

```suggestion
The build completed successfully, but the following processes were still running:
```